### ISSUE TITLE
[Bug] [next] Legacy Story `parameters` is being removed in pre-transform

### ISSUE THREAD

**Issue by xeho91** [storybookjs/addon-svelte-csf#223]
Found by @paoloricciuti [via Discord](https://discord.com/channels/486522875931656193/1093787257397321759/1300499544827625492)

**Input:**

```svelte
<script context="module">
  import { Story } from "@storybook/addon-svelte-csf";
</script>

<Story
  name="Default"
  parameters={{ 
    sveltekit_experimental: {
      stores: {
        page: {
          data: {
            test: 'passed',
          },
        },
        navigating: {
          route: {
            id: '/storybook',
          },
        },
        updated: true,
      },
    },
  }}
>
  <h1>{"Test"}</h1>
</Story>
```

**Output:**

```svelte
<Story name="Default">
  <h1>{"Test"}</h1>
</Story>
```

**Comment by shilman** [comment]
:rocket: Issue was released in [`v5.0.0-next.9`](https://github.com/storybookjs/addon-svelte-csf/releases/tag/v5.0.0-next.9) :rocket:

**Comment by shilman** [comment]
:rocket: Issue was released in [`v5.0.0-next.9`](https://github.com/storybookjs/addon-svelte-csf/releases/tag/v5.0.0-next.9) :rocket:
